JS/Airport.js

```diff
@@ -1,120 +1,117 @@
+      if (militaryPlanes[i].getMilitaryType() == MilitaryType.TYPE_TRANSPORT) {
         transportMilitaryPlanes.push(militaryPlanes[i]);
-        }
-        }
-        return transportMilitaryPlanes;
-    }
-
-
-    getBomberMilitaryPlanes()
-    {
-        let bomberMilitaryPlanes = [];
-        let militaryPlanes = this.getMilitaryPlanes();
-        for (let i = 0; i < militaryPlanes.length; i++) {
-            if (militaryPlanes[i].getMilitaryType()=== MilitaryType.BOMBER) {
-                bomberMilitaryPlanes.push(militaryPlanes[i]);
-            }
-        }
-        return bomberMilitaryPlanes;
-    }
-
-    constructor(planes) {
-        this.planes = planes;
+      }
     }
-
-
-    getExperimentalPlanes() {
-        let experimentalPlanes  = [];
-        this.planes.forEach(plane => {
-            if (plane instanceof experimentalPlane) {//if
-                experimentalPlanes.push(plane);
-            }
-        });
-//return
-        return experimentalPlanes;
-    }
-
-
-
-    sortByMaxDistance() {
-        this.planes.sort((a, b) => (a.Get_Max_Flight_Distance() > b.Get_Max_Flight_Distance()) ? 1 : -1);
-        return this;
-    }
-
-    /**
-     * Sorts by max speed
-     * @return Airport
-     */
-    sortByMaxSpeed() {
-        this.planes.sort((a, b) => (a.getMS() > b.getMS()) ? 1 : -1);
-        return this;
-    }
-
-    sortByMaxLoadCapacity() {
-        this.planes.sort((a, b) => (a.getMinLoadCapacity() > b.getMinLoadCapacity()) ? 1 : -1);
-        return this;
-    }
-
-    getPlanes() {
-        return this.planes;
-    }
-
-
-
-
-    static print(planes) {
-        return JSON.stringify(planes);
+    return transportMilitaryPlanes;
+  }
+
+  getBomberMilitaryPlanes() {
+    let bomberMilitaryPlanes = [];
+    let militaryPlanes = this.getMilitaryPlanes();
+    for (let i = 0; i < militaryPlanes.length; i++) {
+      if (militaryPlanes[i].getMilitaryType() === MilitaryType.BOMBER) {
+        bomberMilitaryPlanes.push(militaryPlanes[i]);
+      }
     }
+    return bomberMilitaryPlanes;
+  }
+
+  constructor(planes) {
+    this.planes = planes;
+  }
+
+  getExperimentalPlanes() {
+    let experimentalPlanes = [];
+    this.planes.forEach((plane) => {
+      if (plane instanceof experimentalPlane) {
+        //if
+        experimentalPlanes.push(plane);
+      }
+    });
+    //return
+    return experimentalPlanes;
+  }
+
+  sortByMaxDistance() {
+    this.planes.sort((a, b) =>
+      a.Get_Max_Flight_Distance() > b.Get_Max_Flight_Distance() ? 1 : -1
+    );
+    return this;
+  }
+
+  /**
+   * Sorts by max speed
+   * @return Airport
+   */
+  sortByMaxSpeed() {
+    this.planes.sort((a, b) => (a.getMS() > b.getMS() ? 1 : -1));
+    return this;
+  }
+
+  sortByMaxLoadCapacity() {
+    this.planes.sort((a, b) =>
+      a.getMinLoadCapacity() > b.getMinLoadCapacity() ? 1 : -1
+    );
+    return this;
+  }
+
+  getPlanes() {
+    return this.planes;
+  }
+
+  static print(planes) {
```

Comment:
This method makes your `Airport` class "magic".
Please remove it and create a separate function helper for that purpose.